### PR TITLE
Add TableContainer as an extension of GridContainer

### PR DESCRIPTION
**Pull request by smoogipoo**
Expands on `GridContainer` to allow customization of columns/headers.

### CODE REVIEW

**Comment by smoogipoo** [comment]
Ignore the CF error?

**Comment by peppy** [comment]
Yeah should be fine.

File: osu.Framework/Graphics/Containers/TableContainer.cs [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+            }
+        }
+
+        /// <summary>
+        /// Adds content to the underlying grid.
+        /// </summary>
+        private void updateContent()
+        {
+            if (content == null)
+                grid.Content = null;
```
Comment (peppy) [inline osu.Framework/Graphics/Containers/TableContainer.cs]: i'd argue showing headers as a default may be best.

File: osu.Framework/Graphics/Containers/TableContainer.cs [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+            }
+        }
+
+        private Dimension rowSize;
+
+        /// <summary>
+        /// Explicit dimensions for rows. The dimension is applied to every row of this <see cref="TableContainer"/>
+        /// </summary>
+        [CanBeNull]
+        public Dimension RowSize
```
Comment (peppy) [inline osu.Framework/Graphics/Containers/TableContainer.cs]: This isn't tested

**Pull request merged**